packages/platform/libraries/private-links.ts

```diff
@@ -0,0 +1,180 @@
+import { generateHashedLink } from "@calcom/lib/generateHashedLink";
+import { isLinkExpired as utilsIsLinkExpired } from "@calcom/lib/hashedLinksUtils";
+import { HashedLinkService } from "@calcom/lib/server/service/hashedLinkService";
+import { HashedLinkRepository } from "@calcom/lib/server/repository/hashedLinkRepository";
+import type { 
+  CreatePrivateLinkInput_2024_06_14,
+  UpdatePrivateLinkInput_2024_06_14
+} from "@calcom/platform-types";
+
+export type PrivateLinkData = {
+  id: string | number;
+  link: string;
+  expiresAt?: Date | null;
+  maxUsageCount?: number | null;
+  usageCount?: number;
+  eventTypeId: number;
+  isExpired: boolean;
+  bookingUrl: string;
+};
+
+
+
+/**
+ * Internal service for managing private links
+ * This wraps the existing HashedLinkService and HashedLinkRepository
+ */
+class PlatformPrivateLinksService {
```

Comment:
Why do we have PlatformPrivateLinksService defined here and in [event-types_2024_06_14/services/private-links.service.ts](https://github.com/calcom/cal.com/pull/22943/files#diff-70e764e515463adac7c8cec307fa265613a9e8761f71de9d18a8b57733f35806) ?